wetterdienst/provider/dwd/mosmix/api.py

```diff
@@ -518,8 +514,6 @@ def _all(self) -> pd.DataFrame:
 
         df = df[(df.iloc[:, 0] != "=====") & (df.iloc[:, 0] != "TABLE") & (df.iloc[:, 0] != "clu")]
```

Comment:
This line is not needed anymore. It appears they cleaned up the file and made it one big CSVish file.

Reply:
Thanks. I've removed it.